disp-reconstruct.py

```diff
@@ -0,0 +1,249 @@
+#!/usr/bin/env python
+
+'''
+
+Example:
+  ./disp-reconstruct.py \
+  --equation=4 \
+  --lookup=./digicampipe/tests/resources/disp_lookup/disp_lookup_method4.npz \
+  --pixels=/home/jakub/science/fzu/sst-1m_simulace/data_test/ryzen_test2018/0.0deg/Data/processed/pixels.txt \
+  --images=/home/jakub/science/fzu/sst-1m_simulace/data_test/ryzen_test2018/0.0deg/Data/processed/events_image_gamma_ze00_az000.txt \
+  --modification \
+  /home/jakub/science/fzu/sst-1m_simulace/data_test/ryzen_test2018/0.0deg/Data/processed/hillas_gamma_ze00_az000.npz
+
+Usage:
+  disp-reconstruct.py [options] <file>...
+
+Options:
+
+  -h --help     Show this screen.
+  -e <int>, --equation=<int>  Equation for the DISP parameter fit. [default: 5]
+  -l <path>, --lookup=<path>  File with lookup table for selected equation.
+  -p <path>, --pixels=<path>  File with pixel coordinates.
+  -a <path>, --images=<path>  File with event images.
+  --modification    Turn on modified DISP method.
+'''
+
+import numpy as np
+from docopt import docopt
+import matplotlib.pyplot as plt
+import matplotlib as mpl
+from matplotlib.patches import Circle
+import events_image
```

Comment:
`import events_image` only works, when one is in the same folder as `events_image.py` when executing `disp-reconstruct.py` but you cannot make assumptions about any users current working directory (CWD).